docs/topics/mta-6-installing-web-console-on-openshift.adoc

```diff
@@ -75,3 +75,42 @@ You can install the {ProductName} ({ProductShortName}) and the {WebName} on Open
 ** *User name*: admin
 ** *Password*: Passw0rd!
 . When prompted, create a new password.
+
+== Memory testing requirements for installing on Red Hat OpenShift Local
```

Comment:
@CarmiWisemon  
Memory requirements for running {ProductShortName} on Red Hat OpenShift Local